conda_build.recipe/meta.yaml

```diff
@@ -16,6 +16,7 @@ build:
 requirements:
   build:
     - python
+    - conda
```

Comment:
@asmeurer All the `build.sh` file did was call `setup.py` and copy the `bdist_conda` file into the correct place.  In a closer to ideal world (i.e., this world: https://github.com/conda/conda/issues/1145) this package does have `conda` as one of the packages it depends, but that can't be done right now so I didn't include it.

So far as I know, this line isn't needed.